addons/sale_timesheet/report/project_report_view.xml

```diff
@@ -6,7 +6,7 @@
             <field name="model">report.project.task.user</field>
             <field name="inherit_id" ref="project.view_task_project_user_pivot"/>
             <field name="arch" type="xml">
-                <xpath expr="//field[@name='total_hours_spent']" position='before'>
+                <xpath expr="//field[@name='remaining_hours']" position='after'>
```

Comment:
Since we will not remove the total hours spent measure shown by default in the pivot, this change is no longer needed. 🙂